run_common_test.erl

```diff
@@ -189,25 +197,25 @@ get_apps() ->
     end.
 
 
-prepare_cover(true) ->
+prepare_cover(Test, true) ->
     io:format("Preparing cover~n"),
-    prepare();
-prepare_cover(_) ->
+    prepare(Test);
+prepare_cover(_, _) ->
     ok.
 
-analyze_coverage(true) ->
-    analyze();
-analyze_coverage(_) ->
+analyze_coverage(Test, true) ->
+    analyze(Test);
+analyze_coverage(_, _) ->
     ok.
 
-prepare() ->
+prepare(Test) ->
     Apps = get_apps(),
-    Compiled = rpc:call(get_ejabberd_node(), mongoose_cover_helper, start, [Apps]),
+    Compiled = rpc:call(get_ejabberd_node(Test), mongoose_cover_helper, start, [Apps]),
     io:format("Compiled modules ~p~n", [Compiled]).
 
-analyze() ->
+analyze(Test) ->
```

Comment:
could you change the export signature for this function  `analyze/0` -> `analyze/1` ?